example/bloc_library/lib/blocs/simple_bloc_delegate.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Flutter Architecture Sample Authors. All rights reserved.
+// Use of this source code is governed by the MIT license that can be found
+// in the LICENSE file.
+
+import 'package:bloc/bloc.dart';
+
+class SimpleBlocDelegate extends BlocDelegate {
```

Comment:
Maybe a quick comment about this class or links to docs? Out of context, it looks a bit weird that you're creating a dummy override then mutating some (perhaps singleton) class with this dummy delegate?

Reply:
✅